examples/braket_features/Getting_Started_with_OpenQASM_on_Braket.ipynb

```diff
@@ -138,15 +138,16 @@
    "source": [
```

Comment:
<p>We are using LocalSimulator() device to run the tasks, but we have instances of SV1 device properties over all notebook cells. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>